docs/quick-reference/index.md

```diff
@@ -37,7 +37,7 @@ This table summarizes the filters and other options available inside a `tasks` b
 | `heading (includes, does not include) <string>`<br>`heading (regex matches, regex does not match) /regex/i`                                                                                         | `sort by heading`                           | `group by heading`     |                        |
 |                                                                                                                                                                                                     |                                             | `group by backlink`    | `hide backlink`        |
 | `description (includes, does not include) <string>`<br>`description (regex matches, regex does not match) /regex/i`                                                                                 | `sort by description`                       |                        |                        |
-| `tag (includes, does not include) <tag>`<br>`tags (include, do not include) <tag>`<br>`tag (regex matches, regex does not match) /regex/i`<br>`tags (regex matches, regex does not match) /regex/i` | `sort by tag`<br>`sort by tag <tag_number>` | `group by tags`        |                        |
+| `has tags`<br>`no tags`<br>`tag (includes, does not include) <tag>`<br>`tags (include, do not include) <tag>`<br>`tag (regex matches, regex does not match) /regex/i`<br>`tags (regex matches, regex does not match) /regex/i` | `sort by tag`<br>`sort by tag <tag_number>` | `group by tags`        |                        |
```

Comment:
Unfortunately this will conflict with a big docs PR I just merged.

When you update, if you haven't done so already, please try to align the `|` column markers for tidiness and ease of future editing. Thanks.

Reply:
I guess this change is easy to do, so I will just have to manage a merge conflict? Or you see another way to go?

Reply:
Yes I think so.